pyvista/utilities/fileio.py

```diff
@@ -274,37 +272,35 @@ def read_meshio(filename, file_format = None):
         vtk_type = meshio_to_vtk_type[k]
         numnodes = vtk_type_to_numnodes[vtk_type]
         offset += [next_offset+i*(numnodes+1) for i in range(len(v))]
-        cells.append(numpy.hstack((numpy.full((len(v), 1), numnodes), v)).ravel())
+        cells.append(np.hstack((np.full((len(v), 1), numnodes), v)).ravel())
         cell_type += [vtk_type] * len(v)
         next_offset = offset[-1] + numnodes + 1
 
         # Extract cell data
         if k in mesh.cell_data.keys():
             for kk, vv in mesh.cell_data[k].items():
                 if kk in cell_data:
-                    cell_data[kk] = numpy.concatenate((cell_data[kk], vv))
+                    cell_data[kk] = np.concatenate((cell_data[kk], vv))
                 else:
                     cell_data[kk] = vv
 
     # Create pyvista.UnstructuredGrid object
+    points = mesh.points
+    if points.shape[1] == 2:
+        points = np.c_[points, np.zeros(len(points))]
```

Comment:
Okay - this appears to have been the needed fix!

Reply:
@keurfonluu, should we handle a 1D case as well?

Reply:
this appears to have broken the existing meshio tests

Reply:
Not this change. It seems to be due to the same type conversion error I had in the following change in ``point_arrays`` and ``cell_arrays``. See below.